deepchecks/vision/utils/detection_encoders.py

```diff
@@ -0,0 +1,64 @@
+from typing import Union, Callable
+
+__all__ = ["DetectionLabelEncoder", "DetectionPredictionEncoder"]
+
+
+class DetectionLabelEncoder:
+    """
+    Class for encoding the detection annotations to the required format.
+
+    Parameters
+    ----------
+    label_encoder : Union[str, Callable]
+        Function or string that specifies the encoding function.
+        If a string, it must be one of the following:
```

Comment:
In the string option - also worth noting that you are supposed to pass a string only if it represents *exactly* the format outputted as a batch from your dataloader. If the match is not exact - use a callable.